homeassistant/components/light/__init__.py

```diff
@@ -447,7 +447,7 @@ def state_attributes(self):
             if supported_features & SUPPORT_COLOR_TEMP:
                 data[ATTR_COLOR_TEMP] = self.color_temp
 
-            if self.supported_features & SUPPORT_COLOR and self.hs_color:
+            if supported_features & SUPPORT_COLOR and self.hs_color:
```

Comment:
Nice catch but this is unrelated to your changes so please remove it from this PR.

Reply:
Ok, I will open a separate PR for that change.